generators/shared/templates/.gitignore.tpl

```diff
@@ -8,10 +8,15 @@
 *.tfstate
 *.tfstate.*
 
+# Go vendor directory
+vendor/
+
+# Module directory
+.test-data/
```

Comment:
What is module directory?

Reply:
`.test-data/` is generated by terratest. "Generated files by terratest" may be a better comment.